org.civicrm.reqangular/src/common/models/instances/contract-instance.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/models-instances',
+  'common/models/instances/instance',
+  'common/services/api/contract'
+], function (instances) {
+  'use strict';
+
+  instances.factory('ContractInstance', ['ModelInstance', 'api.contract',
+    function (ModelInstance, ContractAPI) {
+      return ModelInstance.extend({
+
+        /**
+         * Gets all existing (non-deleted) contracts
+         * @param {Object} params - any additional parameters
+         * @return {Promise}
+         */
+        all: function (params) {
+          return ContractAPI.all(params);
```

Comment:
I don't think I need to extend the Contract at the moment as I don't really need any additional specific methods that can be reused. At the moment. So I just deleted the all() and left a comment.